test/test_transforms.py

```diff
@@ -195,209 +197,486 @@ def test_accimage_crop(self):
+        img_data_byte = torch.ByteTensor(4, 4).random_(0, 255)
+        expected_output = img_data_byte.float().div(255.0).numpy()
+        yield img_data_byte, expected_output, 'L'
 
-    with pytest.raises(TypeError):
-        trans(np.random.rand(1, height, width).tolist())
+        img_data_short = torch.ShortTensor(4, 4).random_()
+        expected_output = img_data_short.numpy()
+        yield img_data_short, expected_output, 'I;16'
 
-    with pytest.raises(TypeError):
-        trans(np.random.rand(1, height, width))
+        img_data_int = torch.IntTensor(4, 4).random_()
+        expected_output = img_data_int.numpy()
+        yield img_data_int, expected_output, 'I'
 
+    @pytest.mark.parametrize('with_mode', [False, True])
+    @pytest.mark.parametrize('img_data, expected_output, expected_mode', _get_1_channel_tensor_various_types())
+    def test_1_channel_tensor_to_pil_image(self, with_mode, img_data, expected_output, expected_mode):
+        transform = transforms.ToPILImage(mode=expected_mode) if with_mode else transforms.ToPILImage()
+        to_tensor = transforms.ToTensor()
 
-def test_randomresized_params():
-    height = random.randint(24, 32) * 2
-    width = random.randint(24, 32) * 2
-    img = torch.ones(3, height, width)
-    to_pil_image = transforms.ToPILImage()
-    img = to_pil_image(img)
-    size = 100
-    epsilon = 0.05
-    min_scale = 0.25
-    for _ in range(10):
-        scale_min = max(round(random.random(), 2), min_scale)
-        scale_range = (scale_min, scale_min + round(random.random(), 2))
-        aspect_min = max(round(random.random(), 2), epsilon)
-        aspect_ratio_range = (aspect_min, aspect_min + round(random.random(), 2))
-        randresizecrop = transforms.RandomResizedCrop(size, scale_range, aspect_ratio_range)
-        i, j, h, w = randresizecrop.get_params(img, scale_range, aspect_ratio_range)
-        aspect_ratio_obtained = w / h
-        assert((min(aspect_ratio_range) - epsilon <= aspect_ratio_obtained and
-                aspect_ratio_obtained <= max(aspect_ratio_range) + epsilon) or
-               aspect_ratio_obtained == 1.0)
-        assert isinstance(i, int)
-        assert isinstance(j, int)
-        assert isinstance(h, int)
-        assert isinstance(w, int)
-
-
-@pytest.mark.parametrize('height, width', [
-    # height, width
-    # square image
-    (28, 28),
-    (27, 27),
-    # rectangular image: h < w
-    (28, 34),
-    (29, 35),
-    # rectangular image: h > w
-    (34, 28),
-    (35, 29),
-])
-@pytest.mark.parametrize('osize', [
-    # single integer
-    22, 27, 28, 36,
-    # single integer in tuple/list
-    [22, ], (27, ),
-])
-@pytest.mark.parametrize('max_size', (None, 37, 1000))
-def test_resize(height, width, osize, max_size):
-    img = Image.new("RGB", size=(width, height), color=127)
-
-    t = transforms.Resize(osize, max_size=max_size)
-    result = t(img)
-
-    msg = "{}, {} - {} - {}".format(height, width, osize, max_size)
```

Comment:
This has been dropped completely as mentioned in [#3952 comment](https://github.com/pytorch/vision/pull/3952#discussion_r645977808)